### PR TITLE
add backlog and dev-strategy to planning

### LINKED ISSUE #1: development strategy and backlog
create development strategy and backlog for the project and share with team for review and comments.

- [X] dev-strategy
- [X] backlog

### PR DESCRIPTION
**Pull request by tahminarasoli**
this pull request will add the modified user story and dev-strategy. according to this project plan team can decide the next steps of the project.
please have a review and let me know your comments, and you are free to bring changes by your self as well.

closes #1

### CODE REVIEW

**Comment by bermarte** [comment]
I am checking it right now.

File: planning/backlog.md [hunk 1]
```diff
@@ -9,36 +9,38 @@ A collection of user stories you can choose from. Stories inside of each priorit
...
 
 - [ ] A user can see all questions
-  - _acceptance criteria ..._
+  - _a list of questions are present in one page. when user choosed to start the quiz there is only one question at a time ..._
 - [ ] A user can select an answer for each question
-  - _acceptance criteria ..._
+  - _questions can be multiple choice or one option. if there was a radio button you have only one choice or if there was a checkbox you have multiple choices ..._
 - [ ] A user can know which questions they got correct and incorrect
-  - _acceptance criteria ..._
+  - _with correct answer you have a green light and for a wrng answer you will see a red color ..._
```
Comment (bermarte) [inline planning/backlog.md:16]: Missing 'wrong'

**Review by bermarte — APPROVED** [review]

**Comment by bermarte** [comment]
I think as soon as a review is done, the reviewer might just merge the PR, otherwise it might take too long.

**Comment by tahminarasoli** [comment]
> 
> 
> I think as soon as a review is done, the reviewer might just merge the PR, otherwise it might take too long.

yes, please go ahead.

**Review by bermarte — APPROVED** [review]

**Comment by bermarte** [comment]
Can you try to rename the folder 'directory'? Something like 'directroy_tmp' or something else. This is to fix an error of the CI. I think we will be able to fix it.

**Comment by bermarte** [comment]
https://www.youtube.com/watch?v=N-PkIszSH3o&feature=youtu.be

**Comment by bermarte** [comment]
Ok, I will try to copy what you did in a new branch with a different name.

**Pull request closed without merging**